packages/formation-react/src/components/CollapsiblePanel/CollapsiblePanel.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: CollapsiblePanel
 name: CollapsiblePanel
+tags: accordion
```

Comment:
i was thinking we may want to group the components under subheadings like 
- progressive disclosures or hiding/showing components (we have a lot of these)
- icons
- inputs
- other?

Reply:
Yeah, that could be useful. Especially for all the forms ones.